README.md

```diff
@@ -4,29 +4,76 @@
 [![Build Status](https://travis-ci.org/jupyter/nbgrader.svg)](https://travis-ci.org/jupyter/nbgrader)
 [![codecov.io](http://codecov.io/github/jupyter/nbgrader/coverage.svg?branch=master)](http://codecov.io/github/jupyter/nbgrader?branch=master)
 
-A system for assigning and grading notebooks.
+A system for assigning and grading Jupyter notebooks.
 
 [Documentation can be found on Read the Docs.](http://nbgrader.readthedocs.org)
 
+
+## Visual highlights of nbgrader
+
+### Instructor toolbar extension for Jupyter notebooks
+The nbgrader toolbar extension for Jupyter notebooks guides the instructor through
+assignment and grading tasks using the familiar Jupyter notebook interface.
+
+![nbgrader toolbar extension for Jupyter notebooks](docs/source/user_guide/images/assignment_toolbar.png "nbgrader toolbar extension for Jupyter notebooks")
+
+![Creating assignment](docs/source/user_guide/images/creating_assignment.gif "Creating assignment")
+
+### Student toolbar extension for Jupyter notebooks
+Using the toolbar extension, students may conveniently view, fetch, submit, and validate their assignments.
+
+![nbgrader assignment list](docs/source/user_guide/images/assignment_list_downloaded_expanded.png "nbgrader assignment list")
```

Comment:
Sorry, I meant you could use the gif I posted earlier for this one instead of the png. Here's the link: https://camo.githubusercontent.com/00fb47c74c41379c1b8ee8f5bcaedb0acb7c7018/687474703a2f2f692e696d6775722e636f6d2f33304a7969615a2e676966